packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1050,4 +1050,18 @@
       <protected>false</protected>
       <shortDescription>People also ask</shortDescription>
     </labels>
+    <labels>
+      <fullName>quantic_InclusionFilter</fullName>
```

Comment:
You should also define a custom label suffixed with `_zero`. Even if a facet should not display a value matching no items, it is possible to get into this situation when loading a search page from a bookmark, for instance.

Reply:
good catch! thank you!